src/Kunstmaan/MenuBundle/AdminList/MenuItemAdminListConfigurator.php

```diff
@@ -38,7 +38,6 @@ public function __construct(EntityManager $em, AclHelper $aclHelper = null, Menu
     public function buildFields()
     {
         $this->addField('title', 'Title', false, 'KunstmaanMenuBundle:AdminList:menu-item-title.html.twig');
-        $this->addField('online', 'Online', false, 'KunstmaanMenuBundle:AdminList:menu-item-online.html.twig');
```

Comment:
An internal menu item will not be shown on the website when the page is offline. So I thinks it is handy to have an indication in the list of menu items, so you directly see the reason why the link is not showing (especially for people with an non-technical background). Perhaps we can leave the column empty for external links if that is the problem..